packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -260,7 +264,7 @@ export function buildRootObject() {
       makeZCFMint,
       makeEmptySeatKit: () => {
         const initialAllocation = harden({});
-        const proposal = harden({ want: {} });
+        const proposal = cleanProposal(getAmountMath, harden({}));
```

Comment:
Semantic change: the proposal needs to be a ProposalRecord (non-optional `want`, `give`, `exit`).